index.html

```diff
@@ -1,10 +1,248 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="css/styles.css" rel="stylesheet" />
-    <title>Document</title>
-  </head>
-  <body></body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="css/styles.css" rel="stylesheet" />
+  <title>Swiss Bagel Salad</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <div class="upper-box">
+      <!-- title -->
+      <div class="title-wrapper background">
+        <div class="swiss">
+          <h1>Swiss</h1>
+        </div>
+        <div class="bagel">
+          <h1>bagel</h1>
+        </div>
+        <div class="salad">
+          <h1>salad</h1>
+        </div>
+      </div>
+      <div class="title-wrapper text">
+        <div class="swiss">
+          <h1>Swiss</h1>
+        </div>
+        <div class="bagel">
+          <h1>bagel</h1>
+        </div>
+        <div class="salad">
+          <h1>salad</h1>
+        </div>
+      </div>
+      <!-- restaurant info -->
+      <div class="info-wrapper">
+        <div class="contact">
+          <ul>
+            <li>Contact</li>
+            <li>Mail</li>
+            <li>Tel.</li>
+          </ul>
+        </div>
+        <div class="content">
+          <ul>
+            <li><a href="swissbagelsalad.ch">swissbagelsalad.ch</a></li>
+            <li><a href="mailto:sbs@bluewin.ch">sbs@bluewin.ch</a></li>
+            <li>022 398 42 66</li>
+          </ul>
+        </div>
+        <div class="social-media">
+          <ul>
+            <li><a href="https://www.facebook.com/">Facebook</a></li>
+            <li><a href="https://www.instagram.com/">Instagram</a></li>
+            <li>2016</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <!-- Menu -->
+    <!-- intro -->
+    <div class="below-box">
+      <div class="intro-wrapper">
+        <div class="intro1">
+          <p>Le bagel facon Suisse</p>
+        </div>
+        <div class="intro2">
+          <p>6 recettes a decouvrir</p>
+        </div>
```

Comment:
Here, you can also eliminate the `div` elements and instead add your class names to the `p` elements:
```suggestion
        <p class="intro1">Le bagel facon Suisse</p>
        <p class="intro2">6 recettes a decouvrir</p>
```

You need the `div` on line 73 because it's wrapping up multiple `p` elements, but the same is not true here.